cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,12 +1,16 @@
 {% set base_metrics_memory = "200Mi" -%}
 {% set metrics_memory = base_metrics_memory -%}
+{% set base_metrics_cpu = "80m" -%}
+{% set metrics_cpu = base_metrics_cpu -%}
 {% set base_eventer_memory = "200Mi" -%}
 {% set eventer_memory = base_eventer_memory -%}
 {% set metrics_memory_per_node = 4 -%}
+{% set metrics_cpu_per_node = 1 -%}
 {% set eventer_memory_per_node = 500 -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
   {% set metrics_memory = (200 + num_nodes * metrics_memory_per_node)|string + "Mi" -%}
+  {% set metrics_cpu = (num_nodes + 80)|string + "m" -%}
```

Comment:
As an update - it's possible to pass 0.5 millicore to the nanny.
However, in nanny it would be treated as 1millicore anyway. We would need to change nanny, which is not trivial, because resource.Quantity (which is used by nanny) doesn't support "multiplication" operation... heh...